app/views/foi/requests/index.html.erb

```diff
@@ -5,7 +5,15 @@
 
 <p class="text">You have a right to request any recorded information held by the Council.</p>
 
-<p class="text">All FOI and EIR responses we send out will be published in the disclosure log on our website.</p>
+<p class="text">
+  <strong class="bold">
+    Find out if your question has been answered before.
+  </strong>
+
+  All FOI and EIR responses we send out will be published in the
+  <%= link_to 'disclosure log', 'https://foi.infreemation.co.uk/hackney/' %> on
```

Comment:
Better link to use might be: https://hackney.gov.uk/foi-disclosure-log

Reply:
Ah yes, definitely!